src/md_lammps.h

```diff
@@ -104,6 +113,19 @@ typedef struct md_lammps_data_t {
 	md_lammps_atom_t* atom_data;
 } md_lammps_data_t;
 
+typedef struct md_lammps_trajectory_t {
```

Comment:
this struct should be inside the translation unit and opaque to the user

Reply:
Was moved to .c